policyengine_us/parameters/gov/states/ny/nysdps/clean_heat/clean_heat_con_edison/multifamily/cap.yaml

```diff
@@ -0,0 +1,11 @@
+description: New York State caps the Clean Heat program for multifamily building residents at this amount.
+metadata:
+  unit: currency-USD
+  period: year
+  label: New York State clean heat multifamily cap
+  reference:
+      - title: New York State Clean Heat Con Edison Heat Pump Program Manual
+        href: https://cleanheat.ny.gov/assets/pdf/CECONY%20Clean%20Heat%20Program%20Manual%206%203%2024.pdf#page=13
+
+values:
+  2024-03-01: 1_000_000
```

Comment:
was the program introduced in 2024?

Reply:
The current incentive amount started from this date. It seems that the program began a bit earlier, but we're not entirely sure when. I looked at an older manual, and found that they had similar but different classification, including limited-time incentives that lasted for a few months. Implementing everything seems to be extremely difficult...

Reply:
Lets stay with the most current manual version then